tasks/task.json

```diff
@@ -41,6 +41,12 @@
             "additionalItems": {
               "type": "string"
             }
+          },
+          "files": {
```

Comment:
We also need this outside the `implementations` array of objects.

Please test out the schema and examples together using something like https://www.jsonschemavalidator.net/.